.github/workflows/test-pypi-publish.yml

```diff
@@ -0,0 +1,75 @@
+name: Test PyPI Publishing
```

Comment:
I'm not sure we want this to run on every version. Issue is that the test PyPi disables accounts on a regular cadence. Lets remove this one. My ask was originally to test it out from your fork against test pypi, not add this to the main fork.
Anyhow, to speed this up, i think we can skip publishing to test pypi for now.